docs/modules/ROOT/pages/libraries.adoc

```diff
@@ -784,3 +784,99 @@ DS.Util.mapToObject(payload, function(x) x.language.name, function(v) v.language
 };
 ------------------------
 
+## Regex
```

Comment:
Title should definitely be Regular Expressions. Also, this is definitely one of those areas we'll want to spend some time later, as non-programmers are not very familiar with regexes